lib/diagnostic.rb

```diff
@@ -91,13 +91,15 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.pop 2' # could use pop, slice, push etc
 
 # Taking the result from the question above, what would `arr` now equal:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
+Diagnostic.array_add_to = '[12, 34, 56, 67, nil, nil, 99]'
+# or (if they're basing it on having popped 2 from the end)
+Diagnostic.array_add_to = '[12, 34, nil, nil, 99]'
```

Comment:
I think you meant to point this to solution too, but since we don't merge on github, that won't affect anything.